Exiled.API/Features/Door.cs

```diff
@@ -53,7 +53,7 @@ public Door(DoorVariant door)
         /// <summary>
         /// Gets or sets a value indicating whether the door is open.
         /// </summary>
-        public bool Open
```

Comment:
Please add the `Open` property back, but mark it as Obsolete.

Reply:
Ok, give me a few minutes

Reply:
That isn't necessary during the alpha stage @PintTheDragon